rust/htp/.gitignore

```diff
@@ -0,0 +1,5 @@
+!Cargo.toml
+/Cargo.lock
+/target
+/vendor
+/gen
```

Comment:
Being part of the work space, these files are never created.

Reply:
You probably need `!Cargo.toml` as its ignored in the `.gitignore` in the parent directory.